src/components/App/App.js

```diff
@@ -0,0 +1,28 @@
+require('normalize.css/normalize.css')
+require('./App.css')
+
+import React from 'react'
+import Navbar from '../Navbar/Navbar'
+import { Switch, Route } from 'react-router-dom'
+import Shop from '../../screens/Shop/Shop'
```

Comment:
hmm that's good idea, but as with everything don't go all the way down the rabbit hole...

Basic don't create `Container Component` (or as you want to named them) for everything, there's no need for some smaller ones just not need noise and it will grow fast!  

I'm calling `container` components `Controllers` (form old times it just stick) so for example you have `Home.jsx` ( page |  container | scene | screen ) that fetch some data and pass it down to
`HomeController.jsx` that mange actions, maybe have some local state, here you include your components and build layout, if this get to complex you can extract from it `HomeLayout.jsx` that will be sliced more etc... you got the idea... it all depends on complexity of layout you are building